lib/levelup.js

```diff
@@ -116,7 +112,12 @@ LevelUP.prototype.open = function (callback) {
   this._status = 'opening'
   this.db      = new DeferredLevelDOWN(this.location)
   dbFactory    = this.options.db || getLevelDOWN()
```

Comment:
this line will throw when you're in the browser and `this.options.db` isn't set, as `getLevelDown === {}`